src/modules/CanvasResize/info.json

```diff
@@ -15,12 +15,12 @@
         "x": {
             "type": "integer",
             "desc": "X-cord of the top left corner of the image on the canvas",
-            "default": 500
+            "default": 200
```

Comment:
Why are these other things different?

Reply:
I don't remember exatcly but maybe it's default value was changed because of this bug or something so I changed it to the original after the bug was fixed(most probably, I don't remeber exactly why I changed this but it was something like this)